### DISCUSSION TITLE
Reply bussiness message

### DISCUSSION THREAD

**Discussion by deulizealand** [mymmrac/telego#178] [Q&A] [answered]
Hi Everyone, i wanna know how to reply to message from user with bot that connected with business account that anounce by telegram ?

i have try to reply with 

`chatId := tu.ID(int64(update.BusinessMessage.Chat.ID))
		msg := tu.Message(chatId, "")
		msg.Text = "Alright " + update.BusinessMessage.From.Username
		bot.SendMessage(msg)`

but error says :
`API response sendMessage: Ok: false, Err: [403 "Forbidden: bot can't initiate conversation with a user"]`

any helps would be appreciate, thank you

**Comment by mymmrac** [comment, marked as answer]
Hello, there are a couple of things:
1. You need to enable Business Mode for your bot in [@BotFather](https://t.me/botfather)
2. Use the `business_connection_id` field in [sendMessage](https://core.telegram.org/bots/api#sendmessage) (or any other send method)

Please refer to docs: https://core.telegram.org/bots/features#bots-for-business

> Note: I personally didn't used Telegram for Business so may not know some nuances
